tests/fixtures/simple-detections/valid/gcp_gcs_public.yml

```diff
@@ -30,22 +30,11 @@ Detection:
       - serviceData
       - policyDelta
       - bindingDeltas
-    Condition: AnyElement
-    Expressions:
-      - Key: action
-        Condition: Equals
-        Value: ADD
-      - Key: member
-        Condition: IsIn
-        Values:
-          - allUsers
-          - allAuthenticatedUsers
-      - Key: role
-        Condition: IsIn
-        Values:
-          - 'roles/storage.objectAdmin'
-          - 'roles/storage.objectViewer'
-          - 'roles/storage.admin'
```

Comment:
why'd this get axed?

Reply:
I believe it is because list comp match conditions don't yet have a python translation

Reply:
Yea, I can pull a different spec if you'd prefer.  I just ran into the backend complaining about the transpilation not existing yet

Reply:
ahh I see ok, no worries